<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>200</red>
          <green>170</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>207</red>
          <green>207</green>
          <blue>207</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>200</red>
          <green>170</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>207</red>
          <green>207</green>
          <blue>207</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>207</red>
          <green>207</green>
          <blue>207</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(207, 207, 207);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>261</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="MENTORMENU">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>521</width>
       <height>111</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>71</red>
           <green>85</green>
           <blue>90</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>71</red>
           <green>85</green>
           <blue>90</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>207</red>
           <green>207</green>
           <blue>207</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>71</red>
           <green>85</green>
           <blue>90</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>71</red>
           <green>85</green>
           <blue>90</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>207</red>
           <green>207</green>
           <blue>207</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>71</red>
           <green>85</green>
           <blue>90</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>207</red>
           <green>207</green>
           <blue>207</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>MENTOR MENU</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(162, 162, 162);
background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_search">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="maxLength">
      <number>32800</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 207, 207);</string>
     </property>
     <property name="text">
      <string>All Conversations</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(207, 207, 207);</string>
     </property>
     <property name="text">
      <string>Back Menu</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>181</y>
       <width>571</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 207, 207);</string>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Görüşme Tarihi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mentinin Adı Soyadı</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mentörün Adı Soyadı</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bilgi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Yönlendirme</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Düşüncesi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Yoğunluk</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Geçmiş</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>141</y>
       <width>561</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 207, 207);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>VIT Projesi Tamamına Katılması Uygun Olur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VIT Projesi ilk IT Eğitimini Almaya Yönlendirilmesi Uygun Olur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VIT Projesi İngilizce Eğitim Almaya Yönlendirilmesi Uygun OLur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VIT Projesi Kapsamında .... Yönlendirilmesi Uygun OLur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Direkt Bireysel Koçluk ile İşe Yönlendirilmesi Uygun Olur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bir Sonraki VIT Projesine Katılması Daha Uygun Olur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Başka Bir Sektöre Yönlendirilmeli</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Diğer</string>
      </property>
     </item>
    </widget>
    <zorder>MENTORMENU</zorder>
    <zorder>lineEdit_search</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>tableWidget</zorder>
    <zorder>comboBox</zorder>
    <zorder>label_2</zorder>
    <zorder>btn_search</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
